x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.test.tsx

```diff
@@ -409,6 +416,72 @@ describe('Response actions history', () => {
       );
     });
 
+    it('should contain download link in expanded row for `get-file` action WITH file operation permission', async () => {
+      const privileges = useUserPrivilegesMock();
+
+      useUserPrivilegesMock.mockImplementationOnce(() => {
+        return {
+          ...privileges,
+          endpointPrivileges: {
+            ...privileges.endpointPrivileges,
+            canWriteFileOperations: true,
```

Comment:
you don't have to mock user privileges to `true` - that is the default on this module ([its mocked](https://github.com/elastic/kibana/blob/6efef0496077f4e61d49e4e43f75941ff3d98d9e/x-pack/plugins/security_solution/public/common/components/user_privileges/__mocks__/index.ts) as soon as you call `jest.mock()`)

Reply:
72aed7e3c97db58552342377db11fd361b4ffaa7